crates/shadowsocks-service/src/local/redir/udprelay/sys/unix/bsd.rs

```diff
@@ -288,11 +335,13 @@ fn set_socket_before_bind(addr: &SocketAddr, socket: &Socket) -> io::Result<()>
     set_ip_origdstaddr(level, socket)?;
 
     // 3. disable IP fragmentation
+    #[cfg(not(target_os = "openbsd"))]//seems unsupported in OpenBSD
```

Comment:
Only FreeBSD-like system supports IP_DONTFRAG, which are:

```rust
#[cfg(any(target_os = "freebsd",
                  target_os = "dragonfly",
                  target_os = "macos",
                  target_os = "ios",
                  target_os = "watchos",
                  target_os = "tvos"))]
```

Reply:
Maybe we should separate them into 3 files:

1. macos.rs
2. freebsd.rs (freebsd-like)
3. openbsd.rs (netbsd-like)

Reply:
That's an idea, but some functions are used from both OSes. Can I copy&paste them?